conversions/temperature_conversions.py

```diff
@@ -26,6 +26,10 @@ def celsius_to_fahrenheit(celsius: float, ndigits: int = 2) -> float:
     ...
     ValueError: could not convert string to float: 'celsius'
     """
+    if type(celsius)!=float:
+        raise ValueError ("You have entered a string.Enter the value of celsisus to be converted in numbers only.")  #exception handling
```

Comment:
Please read the definition of TypeError vs. ValueError the Python docs.  These should be TypeErrors.

Reply:
Okay sir.But kindly review my other contribution. I understood it and thus added another whole new temperature called reaumur.It is not yet reviewed.Probably I think I got blocked.I am new to this.I did'nt know about that Gap checks , (the one which checks whether 2 line space is given , etc) And instead of committing minor changes,  without knowing I did a new pull request everytime.Now I learnt what it is and how to do.Kindly check my new addition.I think they misunderstood me as a spammer. But i gave so many minor PR without knowing we have an option to edit changes in same PR